lambda/translcore.ml

```diff
@@ -220,7 +220,7 @@ let rec push_defaults loc bindings use_lhs arg_mode cases partial warnings =
             ({exp with exp_type = pat.pat_type; exp_env = env; exp_desc =
               Texp_ident
                 (Path.Pident param, mknoloc (Longident.Lident name),
-                 desc, Id_value)},
+                 desc, Id_value, (Mode.Uniqueness.shared, Mode.Linearity.many))},
```

Comment:
I think this mode will never be consumed, but using `mode` here would be more correct than this dummy value.